From ea261e84ac17ee085fc1b7b5b5eb5e1e44556b83 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Fri, 8 Jan 2021 03:27:07 +0900
Subject: [PATCH 75/99] (DO NOT MERGE) ODROID-COMMON: drm/meson: not load
 RGB709 to YUV709 coefficient

By loading this matrix, OSD color becomes a bit reddish.

Signed-off-by: Dongjin Kim <tobetter@gmail.com>
Change-Id: Ice08001c967c39b638ab534453b784c2958ff7f0
---
 drivers/gpu/drm/meson/meson_viu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/meson/meson_viu.c b/drivers/gpu/drm/meson/meson_viu.c
index bb7e109534de..c23efae7dee2 100644
--- a/drivers/gpu/drm/meson/meson_viu.c
+++ b/drivers/gpu/drm/meson/meson_viu.c
@@ -425,6 +425,7 @@ void meson_viu_init(struct meson_drm *priv)
 	if (meson_vpu_is_compatible(priv, VPU_COMPATIBLE_GXM) ||
 	    meson_vpu_is_compatible(priv, VPU_COMPATIBLE_GXL))
 		meson_viu_load_matrix(priv);
+#if 0
 	else if (meson_vpu_is_compatible(priv, VPU_COMPATIBLE_G12A)) {
 		meson_viu_set_g12a_osd1_matrix(priv, RGB709_to_YUV709l_coeff,
 					       true);
@@ -433,6 +434,7 @@ void meson_viu_init(struct meson_drm *priv)
 				OSD1_HDR2_CTRL_VDIN0_HDR2_TOP_EN, 0,
 				priv->io_base + _REG(OSD1_HDR2_CTRL));
 	}
+#endif
 
 	/* Initialize OSD1 fifo control register */
 	reg = VIU_OSD_DDR_PRIORITY_URGENT |
-- 
2.37.3

